Gemfile

```diff
@@ -3,4 +3,6 @@
 source 'https://rubygems.org'
 gem 'xcpretty'
 gem 'cocoapods'
-gem 'fastlane'
\ No newline at end of file
+gem 'fastlane'
+plugins_path = File.join(File.dirname(__FILE__), 'fastlane', 'Pluginfile')
```

Comment:
What a weird way to go. I guess Ruby folks gonna Ruby.

Reply:
Haha yep, autogenerated courtesy of Fastlane